dbms/src/Encryption/MockKeyManager.cpp

```diff
@@ -107,6 +107,10 @@ FileEncryptionInfo MockKeyManager::getFile(const String & fname)
     std::ignore = fname;
     if (encryption_enabled)
     {
+        if (!fileExist(fname))
+        {
+            files.emplace_back(fname);
+        }
```

Comment:
In fact, the creation of BlobFile does not pass this parameter. Because for BlobFile there is no way to know if it is a newly created or old one. I would think that this is a relatively safe approach so that the proxy can make its own judgment.